arch/x86/deflate_quick.c

```diff
@@ -32,7 +32,7 @@
 extern void fill_window_sse(deflate_state *s);
 extern void flush_pending(PREFIX3(stream) *strm);
 
-static inline long compare258(const unsigned char *const src0, const unsigned char *const src1) {
+ZNG_TARGET_SSE2 ZNG_TARGET_SSE42 static inline long compare258(const unsigned char *const src0, const unsigned char *const src1) {
```

Comment:
This is wrong. Since SSE2 is a subset of SSE4.2, you only need to specify SSE4.2.
(That is also why you don't need to do "-msse2 -msse42")

Reply:
Ok. This should be fixed now.